pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[project]
+name = "cleanmail"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
please borrow the description from the readme.md

`A tool to quickly clean up your email inbox!`